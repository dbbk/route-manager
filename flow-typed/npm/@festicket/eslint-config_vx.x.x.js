// flow-typed signature: 564086c0274f153b4c37ac5c954f219e
// flow-typed version: <<STUB>>/@festicket/eslint-config_v^1.1.5/flow_v0.63.1

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@festicket/eslint-config'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@festicket/eslint-config' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */


// Filename aliases
declare module '@festicket/eslint-config/index' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
declare module '@festicket/eslint-config/index.js' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
