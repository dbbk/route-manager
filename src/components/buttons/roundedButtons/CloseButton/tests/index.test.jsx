// @flow

import React from 'react';
import { RoundedCloseButton } from '@festicket/react-ui-components';
import { renderWithTheme } from 'src/utils/testing/index';

describe('<RoundedCloseButton />', () => {
  test('as button', () => {
    const tree = renderWithTheme(<RoundedCloseButton element="button" />);
    expect(tree).toMatchSnapshot();
  });
  test('as anchor', () => {
    const tree = renderWithTheme(<RoundedCloseButton element="a" />);
    expect(tree).toMatchSnapshot();
  });
});
