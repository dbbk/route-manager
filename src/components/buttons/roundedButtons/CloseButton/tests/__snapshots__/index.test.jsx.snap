// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RoundedCloseButton /> as anchor 1`] = `
.c3 {
  height: 18px;
  fill: #A2A4AD;
}

.c1 {
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #8C64CB;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 38px;
  height: 38px;
}

.c0 {
  display: inline-block;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  border-radius: 50%;
  padding: 0;
  cursor: pointer;
  background-color: white;
  box-shadow: 0 2px 30px 0 rgba(36,37,41,0.15);
}

<a
  className="c0 c1"
  element="a"
  target="_blank"
  variant="standard"
  weight="bold"
>
  <div
    className="c2"
  >
    <svg
      className="c3"
      color="grey"
      viewBox="0 0 32 32"
    >
      <path
        d="M18.507 16l9.404-9.404a1.778 1.778 0 0 0-2.51-2.505l-9.402 9.402-9.404-9.422a1.785 1.785 0 1 0-2.524 2.524l9.422 9.404-9.422 9.404a1.778 1.778 0 1 0 2.505 2.509L16 18.505l9.404 9.404a1.778 1.778 0 0 0 2.505-2.51z"
      />
    </svg>
  </div>
</a>
`;

exports[`<RoundedCloseButton /> as button 1`] = `
.c3 {
  height: 18px;
  fill: #A2A4AD;
}

.c1 {
  border: none;
  outline: none;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 38px;
  height: 38px;
}

.c0 {
  display: inline-block;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  border-radius: 50%;
  padding: 0;
  cursor: pointer;
  background-color: white;
  box-shadow: 0 2px 30px 0 rgba(36,37,41,0.15);
}

<button
  aria-disabled={undefined}
  className="c0 c1"
  disabled={undefined}
  size="regular"
>
  <div
    className="c2"
  >
    <svg
      className="c3"
      color="grey"
      viewBox="0 0 32 32"
    >
      <path
        d="M18.507 16l9.404-9.404a1.778 1.778 0 0 0-2.51-2.505l-9.402 9.402-9.404-9.422a1.785 1.785 0 1 0-2.524 2.524l9.422 9.404-9.422 9.404a1.778 1.778 0 1 0 2.505 2.509L16 18.505l9.404 9.404a1.778 1.778 0 0 0 2.505-2.51z"
      />
    </svg>
  </div>
</button>
`;
